src/Expander/Primitives.hs

```diff
@@ -45,25 +45,22 @@ module Expander.Primitives
   , syntaxError
   , the
   , typeCase
-  -- * Type primitives
-  , arrowType
-  , baseType
-  , macroType
-  , ioType
-  , primitiveDatatype
   -- * Pattern primitives
   , elsePattern
   -- * Module primitives
   , makeModule
   -- * Anywhere primitives
   , makeLocalType
-  -- * Local primitives
-  , prepareVar
```

Comment:
I don't know what's a "local primitive", but `prepareVar` isn't exposed to the kernel, so I wouldn't call it a primitive.